azurepipelines/e2e_test/e2etest.yaml

```diff
@@ -130,7 +130,7 @@ stages:
                       du_tarball_script: >-
                           tar -xvf /tmp/testsetup.tar.gz -C ./ &&
                           chmod u=rwx,g=rwx,o=rx ./testsetup/setup.sh &&
-                          ./testsetup/setup.sh --self-upgrade
+                          ./testsetup/setup.sh --distro ubuntu-18.04 --architecture amd64
```

Comment:
what happened to '--self-upgrade' option?

Reply:
I have a separate PR that add supports self upgrade , removing it here for the simplicity of this PR